### ISSUE TITLE
No app in context warning

### ISSUE THREAD

**Issue by paulquirk** [nextcloud/social#492]
My Nextcloud log shows dozens of these warnings every day, each one from a different IP address running Social 0.1.4:
----------------------------------------------------------------------------------------------------------
[no app in context] Warning: 500 - {"status":-1,"exception":"OCA\\Social\\Exceptions\\InvalidOriginException","message":""}

POST /index.php/apps/social/inbox
from 136.243.88.169 at 2019-04-16T05:09:19+00:00
---------------------------------------------------------------------------------------------------------------------
Operating system: Linux 4.15.0-47-generic #50-Ubuntu SMP Wed Mar 13 10:44:52 UTC 2019 x86_64

Webserver: Apache/2.4.29 (Ubuntu) (apache2handler)

Database: mysql 10.1.38

PHP version:

7.1.28-1+ubuntu18.04.1+deb.sury.org+3 Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, apache2handler, mysqlnd, PDO, xml, apcu, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, igbinary, imagick, intl, json, exif, mcrypt, mysqli, pdo_mysql, apc, posix, readline, redis, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlrpc, xmlwriter, xsl, zip, Phar, Zend OPcache

Nextcloud version: 15.0.7 - 15.0.7.0

**Comment by daita** [comment]
Hello,

You should have some logs before this one showing the incoming request. would you mind sharing one of them ?

**Comment by thecyberd3m0n** [comment]
I have same problem
```
{"reqId":"mEMaLqXTT4bcAO6XRuEO","level":2,"time":"2019-04-25T08:48:28+00:00","remoteAddr":"192.168.1.1","user":"ohm0n","app":"no app in context","method":"GET","url":"\/apps\/social\/api\/v1\/global\/account\/info?account=ohm0n@tcdintranet.icu","message":"500 - {\"status\":-1,\"exception\":\"OCA\\\\Social\\\\Exceptions\\\\InvalidOriginException\",\"message\":\"\"}","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/74.0.3729.108 Safari\/537.36","version":"15.0.7.0"}
```
when trying to configure Social app

my host is https://tcdintranet.icu

**Comment by daita** [comment]
@thecyberd3m0n please create a new report with the value of:

>      ./occ config:app:get social address

**Comment by thecyberd3m0n** [comment]
[admin@TCDCLOUD001 ~]# occ config:app:get social address
https://tcdintranet.icu

**Comment by thecyberd3m0n** [comment]
any further suggestions? Should I change something to make it work?

**Comment by susobaco** [comment]
Hello, I have the same error, with the same command output: occ config:app:get social address
https://palautgn.duckdns.org  (my domain)

My system is debian stretch with nginx/1.17.0, PHP 7.3.5, MariaDB-1:10.3.15, Nextcloud 16.0.1 ,Social 0.2.2

**Comment by daita** [comment]
should be fixed in 0.2.6

**Comment by susobaco** [comment]
Yes, thank you.
